### PR TITLE
 Add initial setup for react-redux project

### PR DESCRIPTION
**Pull request by codecaiine**
- Initialed  Setup for our react-redux Space Travelers website!
- Created React App using React-Redux template
-  Installed React-Redux, Redux Logger, and React Router

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
-# space-travelers-hub
\ No newline at end of file
+# Getting Started with Create React App
```
Comment (Isabellanamutenda) [inline README.md]: This should be deleted

**Review by Isabellanamutenda — COMMENTED** [review]
Good job on the work so far :+1: 
Kindly make the changes on the review

File: README.md (hunk 1 above)
Comment (codecaiine) [reply README.md]: Thanks isabella @Isabellanamutenda

**Review by Isabellanamutenda — APPROVED** [review]
Project approved

**Pull request merged**